dbms/src/Functions/tests/bench_collation.cpp

```diff
@@ -0,0 +1,75 @@
+// Copyright 2022 PingCAP, Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <Functions/FunctionsComparison.h>
+#include <TestUtils/FunctionTestUtils.h>
+
+#include <benchmark/benchmark.h>
+#include <random>
+
+namespace DB
+{
+namespace tests
+{
+
+class CollationBench : public benchmark::Fixture
+{
+public:
+    void SetUp(const benchmark::State &) override {
+        data.push_back(toVec<String>("col1", std::vector<ColStringType>(1000000, "aaaaaaaaaaaaa")));
```

Comment:
maybe add more data?

Reply:
> 

It's enough to show performance difference between collations.

Reply:
Can you show the current performance result of different collations?

Reply:
> Can you show the current performance result of different collations in this PR?

okk